tracer/test/snapshots/AwsSnsTests.NetCore.SchemaV0.verified.txt

```diff
@@ -75,15 +65,62 @@
       http.url: http://localhost:00000/,
       runtime-id: Guid_1,
       span.kind: client,
-      topicname: MyTopic,
-      _dd.p.dm: -0
+      topicname: MyTopic
     },
     Metrics: {
-      process_id: 0,
-      _dd.agent_psr: 1.0,
-      _dd.top_level: 1.0,
-      _dd.tracer_kr: 1.0,
-      _sampling_priority_v1: 1.0
+      _dd.top_level: 1.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_5,
+    Name: sns.request,
+    Resource: SNS.PublishBatch,
```

Comment:
I expected to see `Publish` spans too, but I only see `PublishBatch` 🤔 Also, we should probably test both the async and non-async methods too (where possible), seeing as there's an integration for them here 🙂